e2e/testcases/custom_resource_definitions_test.go

```diff
@@ -118,13 +118,10 @@ func mustRemoveCustomResourceWithDefinition(nt *nomostest.NT, crd client.Object)
 }
 func TestMustRemoveCustomResourceWithDefinitionV1Beta1(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.Reconciliation1)
-	support, err := nt.SupportV1Beta1CRDAndRBAC()
-	if err != nil {
-		nt.T.Fatal("failed to check the supported CRD versions")
-	}
-	if support {
-		mustRemoveCustomResourceWithDefinition(nt, anvilV1Beta1CRD())
+	if !nt.SupportV1Beta1CRDAndRBAC() {
+		nt.T.Skip("Kubernetes v1.22 and later do not support the v1beta1 CRD API")
```

Comment:
v1.22 and below is no longer in our testing matrix, so we could also consider removing these tests

Reply:
Suggest filing a bug for that.